windows/src/desktop/kmshell/install/uninstall.pas

```diff
@@ -165,7 +165,7 @@ function UninstallKeyboard(Owner: TWinControl; const KeyboardName: WideString; S
   kbd.Uninstall;
   kbd := nil;
   kmcom.Keyboards.Refresh;
-  kmcom.Keyboards.Apply;
+  kmcom.Apply;
```

Comment:
Yet here, we use `kmcom.Apply` over `kmcom.Keyboards.Apply`, despite using `kmcom.Keyboards.Refresh`?

Reply:
Tricksy, isn't it. We can do a full `kmcom.Refresh` / `kmcom.Apply` here, which would be better. See also lines 157-158 which should really be the same. I'll fix those up.

Reply:
Eventually, I want to refactor all this to stop using a global `kmcom` and instantiate the class locally at each change. But that's a much bigger job with potential performance side-effects and mucho testing required, so we'll live with some of the unnecessary stuff such as the `Refresh`+`Apply` pattern which probably only needs to be an `Apply` pattern when making changes on a local `kmcom` object and its members.